MobiusCore/Source/EffectHandlers/EffectHandler.swift

```diff
@@ -28,13 +28,16 @@ public protocol EffectHandler {
 
     /// Handle an `Effect`.
     /// To output events, call `callback.send`.
-    /// When you are done handling `input`, be sure to call `callback.end()` to prevent memory leaks.
-    /// If it does not make sense to finish handling an effect, you should be using a `Connectable` instead of this protocol.
+    /// Call `callback.end()` once the effect has been handled to prevent memory leaks.
     ///
+    /// This returns a `Disposable` which tears down any resources that is being used by this effect handler. This `Disposable` will
```

Comment:
```suggestion
    /// This returns a `Disposable` which tears down any resources used to handle this effect. This `Disposable` will
```

Reply:
And maybe it's not even about tearing down resources, but cancelling the handling of this effect?